stestr/subunit_runner/run.py

```diff
@@ -84,7 +84,7 @@ def _list(self, test):
 
 def main():
     runner = SubunitTestRunner
-    if sys.version_info[0] >= 3:
+    if sys.version_info[0] >= 3 and sys.version_info[1] >= 5:
```

Comment:
It might be better to add a comment/note to understand the magic number '5' here. But I think it's not a blocker for merging because we can check it with git log/blame.